index.js

```diff
@@ -318,6 +320,24 @@ class ServerlessCustomDomain {
       throw new Error(`Cannot find specified domain name ${this.givenDomainName}.`);
     });
   }
+
+  /**
+   * Prints out a summary of all domain manager related info
+   */
+  domainSummary() {
+    this.getDomain().then((data) => {
+      this.serverless.cli.consoleLog(chalk.yellow.underline('Serverless Domain Manager Summary'));
+      if (this.serverless.service.custom.customDomain.createRoute53Record !== undefined
+          && this.serverless.service.custom.customDomain.createRoute53Record === true) {
+        this.serverless.cli.consoleLog(chalk.yellow('Domain Name'));
```

Comment:
Nope, `serverless.cli.log` prints in yellow, but it also appends "Serverless:" to the beginning of the message specified.